pkg/test/step.go

```diff
@@ -355,14 +356,28 @@ func (s *Step) CheckResourceAbsent(expected runtime.Object, namespace string) er
 	return nil
 }
 
+// CheckAssertCommands checks if the commands defined in TestAsserts have been run successfully
+// the errors returned can be a a failure of executing the command or the failure of the command executed.
+func (s *Step) CheckAssertCommands(ctx context.Context, namespace string, command []harness.TestAssertCommand, timeout int) []error {
+	testErrors := []error{}
+	if _, err := testutils.RunAssertCommands(ctx, s.Logger, namespace, command, "", timeout); err != nil {
+		testErrors = append(testErrors, err)
+	}
+	return testErrors
+}
+
 // Check checks if the resources defined in Asserts and Errors are in the correct state.
```

Comment:
let's also not that we run commands now

Reply:
I am not sure what you ment here. I've updated the comment as what I read that the comment was not explaining that we also run the commands here.